data/templates/Detector_StaticSites.csx

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+		let startTime = datetime({cxt.StartTime});
+		let endTime = datetime({cxt.EndTime});
+		YOUR_TABLE_NAME
+		| where TIMESTAMP >= startTime and TIMESTAMP <= endTime
+		YOUR_QUERY
+	";
+}
+
+
+[ArmResourceFilter(provider: "Microsoft.Web", resourceTypeName: "staticSites")]
+[Definition(Id = "YOUR_DETECTOR_ID", Name = "", Author = "YOUR_ALIAS", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<ArmResource> cxt, Response res)
+{
+    res.Dataset.Add(new DiagnosticData()
+    {
+        Table = await dp.Kusto.ExecuteQuery(GetQuery(cxt), cxt.Resource.Stamp.Name, null, "GetQuery"),
```

Comment:
cxt.Resource will not have Stamp property here

Reply:
I suggest in the template, we can add a line to fetch the stamp name via observer

Reply:
From what I learned, staticwebsites does not have a concept of stamp. Our infrastructure where we determine kusto cluster by stampname will not apply for staticwebsites.

Reply:
Let's confirm with Mitch then first, thank you all.